training_pipeline/apps/gradio/semantic_segmentation/app.py

```diff
@@ -1,255 +1,95 @@
-import gradio as gr
+import os
 
-from matplotlib import gridspec
-import matplotlib.pyplot as plt
+import gradio as gr
 import numpy as np
-from PIL import Image
 import tensorflow as tf
-from transformers import SegformerFeatureExtractor, TFSegformerForSemanticSegmentation
+from huggingface_hub import from_pretrained_keras
+from PIL import Image
+
+MODEL_REPO_ID = os.environ("MODEL_REPO_ID")
+MODEL_VERSION = os.environ("MODEL_VERSION")
+MODEL_CKPT = f"{MODEL_REPO_ID}@${MODEL_VERSION}"
+MODEL = from_pretrained_keras(MODEL_CKPT)
```

Comment:
sorry for the misleading information. these keywords are not replaced with environment variables. they are just keywords, so any unique strings can be used. 

Hence, the following lines can be removed.
```
MODEL_REPO_ID = os.environ("MODEL_REPO_ID")
MODEL_VERSION = os.environ("MODEL_VERSION")
```

Also, `MODEL_CKPT` should be replaced like below:
```python
MODEL_CKPT = "$MODEL_REPO_ID@$MODEL_VERSION"
```